AC-Meeting-Guidelines.md

```diff
@@ -0,0 +1,38 @@
+# Architecture Committee Meeting Guidelines
+
+The Architecture Committee (AC) recommends following a set of guidelines for raising issues and adding topics to the meeting agenda.
+
+## Use a GitHub issue or an email to the mailing list first
+
+For a project like Kata Containers, with contributors spanning across so many different time zones, asynchronous discussions and reviews are the preferred communication media.
+
+Directly starting a proposal or trying to resolve new technical issues through the AC meeting has many drawbacks:
+1. It potentially leaves a large part of the community out of the initial dicussion.
+1. It does not give the AC meeting audience enough time to think through the proposal and give constructive and well thought feedback.
+1. It makes it hard to track and share the discussion outcome.
+
+As a consequence, the AC recommends using GitHub issues and PRs, or the kata-dev mailing list at first before adding new items to the AC meetings agenda, especially if you think the topic could be discussed and resolved in less than two weeks.
+
+## AC Agenda topics must be supported by a GitHub issue or PR, or an email sent to the mailing list
+
+Except for a few cases listed in the next section, any topic brought to the AC meeting must be linked to a GitHub issue, PR, or to an email to the mailing list.
+Unless the topic must be urgently discussed, the AC recommends waiting for one or two weeks before bringing the discussion to the AC meeting. That will give enough time for interested contributors to digest the issue or proposal before having a live discussion in the AC meeting.
+ 
+
+### Exceptions
+
+* Announcements
+* Presentations
+* Call for reviews of articles, blog posts, etc.
+
+## AC agenda topics should have well defined expectations
+
+When adding a topic to the agenda, one should have clearly defined expectations: **"What do I want to get from the community and the AC with that discussion?"**.
+
+For instance:
+* Are you looking for a possible **solution** to a problem exposed throug GitHub or the mailing list?
+* Are you looking for a **resolution** of a conflict in a pull request?
+* Are you looking for a **"go-nogo"** from the community in order to start working on something?
+* Are you looking for technical **feedback and guidelines** on a pending technical proposal?
```

Comment:
Good list. Would be nice to add it in an issue template to make sure this is captured.

Also, I would mention possible timing constraints, like "do you have a deadline".

Reply:
> Good list. Would be nice to add it in an issue template to make sure this is captured.

I am not expecting folks to open an issue for adding an item to the agenda. The message here is that if you bring something to the meeting (a feature proposal, a long term issue, etc) it should be described through a regular issue first and then potentially discussed in the AC.